courses/fundamentals_of_ada/adv_140_access_types.rst

```diff
@@ -197,11 +296,38 @@ Introduction to Accessibility Checks
          end Proc;
       end P;
 
-* Access types can access objects at most of the same depth
+* Access types can access objects of the **same or lower** depth
 * The compiler checks it statically
```

Comment:
"lower" is confusing - you don't mean "deeper", you mean "lower number"

Reply:
Yes, and added an example